NeerajChaudhary.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Neeraj Chadhary",
+  "skills": ['learning'],
+  "cause": "the community",
```

Comment:
Atleast, add  github link, if you dont' have other handles.

Reply:
Ok I add my GitHub link as well